package.json

```diff
@@ -20,24 +20,38 @@
     "url": "https://github.com/metarhia/globalstorage"
   },
   "main": "gs.js",
+  "browser": {
+    "gs.js": "dist/gs.js",
+    "lib/fs.cursor.js": false,
+    "lib/fs.provider.js": false
+  },
   "scripts": {
     "test": "npm run lint && node test/all",
-    "lint": "eslint ."
+    "lint": "eslint . --ignore-path .gitignore",
+    "build": "babel gs.js -d dist && babel lib -d dist/lib --ignore lib/fs.provider.js,lib/fs.cursor.js",
+    "prepublish": "npm run -s build"
   },
   "engines": {
     "node": ">=6.0.0"
   },
   "dependencies": {
+    "@babel/runtime": "^7.0.0",
```

Comment:
Keep in mind that it makes `@babel/runtime` a dependency of the package, and it won't be used much of the time (i.e., in Node.js). Can you please compare the build size with and without `@babel/plugin-transform-runtime`?

Reply:
It's 64K vs 68K.